app/views/restos/show.html.erb

```diff
@@ -42,13 +46,89 @@
+
+  <!-- Add this css in head tag -->
+
+
+
+     <!-- Here is the section -->
 
+      <section id="counter" class="counter">
+            <div class="main_counter_area">
+                <div class="overlay p-y-3">
+                    <div class="container">
+                        <div class="row">
+                            <div class="main_counter_content text-center white-text wow fadeInUp">
+                                <div class="col-md-3">
+                                    <div class="single_counter p-y-2 m-t-1">
+                                        <i class="fas fa-leaf m-b-1"></i>
+                                        <br>
+                                        <p>Vegan</p>
+                                        <h2 class="statistic-counter"><%= @num_vegan_votes %></h2>
+
+                                    </div>
+                                </div>
+                                <div class="col-md-3">
+                                    <div class="single_counter p-y-2 m-t-1">
+                                        <i class="fas fa-dove m-b-1"></i>
+                                        <br>
+                                        <p>Végétariens</p>
+                                        <h2 class="statistic-counter"><%= @num_veggie_votes %></h2>
+
+                                    </div>
+                                </div>
+                                <div class="col-md-3">
+                                    <div class="single_counter p-y-2 m-t-1">
+                                        <i class="fab fa-pagelines m-b-1"></i>
+                                        <br>
+                                        <p>Gluten-free</p>
+                                        <h2 class="statistic-counter"><%= @num_gluten_votes %></h2>
+
+                                    </div>
+                                </div>
+                                <div class="col-md-3">
+                                    <div class="single_counter p-y-2 m-t-1">
+                                        <i class="fa fa-globe m-b-1"></i>
+                                        <br>
+                                        <p>Au total</p>
+                                        <h2 class="statistic-counter"><%= @num_votes %></h2>
+
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section><!-- End of counter Section -->
+
+
+<!-- Add this script before </body> -->
+
+
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/waypoints/4.0.1/jquery.waypoints.min.js"></script>
+        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Counter-Up/1.0.0/jquery.counterup.min.js"></script>
+
+<script type="text/javascript">
+  jQuery('.statistic-counter').counterUp({
```

Comment:
Why this code is duplicated (in `show` and in `application.js`)?

Reply:
Was not sure it was working in `application.js` so put it in `show` and forgot to delete it Fromm `application.js`